models/command.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Command = require('../config/command');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .description('Identifier of this command')
+        .example(123345),
+
+    labels: Joi
```

Comment:
What is `labels`? [design doc](https://github.com/screwdriver-cd/screwdriver/blob/master/design/commands.md) does not say about this.

Reply:
I add labels (tags) like the existing `template` schema.
https://github.com/screwdriver-cd/data-schema/blob/master/models/template.js
I guess that get api should return its command details with labels (tags).